packages/inventory/src/Stocktake/DetailView/SidePanel.tsx

```diff
@@ -35,17 +47,6 @@ const AdditionalInfoSection: FC<SidePanelProps> = ({ draft }) => {
   );
 };
 
-const RelatedDocumentsSection: FC<SidePanelProps> = () => {
```

Comment:
inventory adjustments created is a neat idea. We've somewhat skipped over stocktakes in the design phase so we have no guidance from that group yet. I'll ask about what to include here.